pkg/cmd/configs.go

```diff
@@ -61,9 +61,19 @@ var configsDeleteCmd = &cobra.Command{
 var configsUpdateCmd = &cobra.Command{
 	Use:               "update [config]",
 	Short:             "Update a config",
+	Long:              "Update properties about a config, such as its name, inheritability flag, and inheritances",
 	Args:              cobra.MaximumNArgs(1),
 	ValidArgsFunction: configNamesValidArgs,
 	Run:               updateConfigs,
+	Example: `Updating a config's name
+$ doppler configs update --project proj --config dev_branch --name dev_branch2
+
+Enabling a config to be inherited
+$ doppler configs update --project proj --config dev --inheritable true
+
+Configuring which configs the given config inherits
+Note: The inherits flag accepts a comma separated list of PROJ_NAME.CONF_NAME
+$ doppler configs update --project proj --config dev --inherits "shared-db.dev,shared-api.dev"`,
```

Comment:
There doesn't seem to be a way to clear the inherits list.

This was the first thing that came to mind for me to try but I'm not sure if this library can distinguish between this and an omitted flag:
```
$ doppler configs -p weather-service update -c dev --inherits ""
Error: flag needs an argument: --inherits
```

An alternative might be to have a sentinel value like `--inherits none`?

Reply:
To clear the inherits list, you can pass in `--inherits=`. I'll add some documentation.